meteor/client/ui/SegmentTimeline/SegmentTimelineContainer.tsx

```diff
@@ -361,6 +363,30 @@ export const SegmentTimelineContainer = withTracker<IProps, IState, ITrackedProp
 		}
 	}
 
+	onGoToPart = (e: CustomEvent<IGoToPartEvent>) => {
+		if (this.props.segmentId === e.detail.segmentId) {
+			for (const part of this.props.parts) {
```

Comment:
Wouldn't this be easier to read with a `this.props.parts.find((part) => part.partId === e.detail.partId)` ?